src/main/java/gyro/aws/ec2/PlacementGroupFinder.java

```diff
@@ -0,0 +1,71 @@
+package gyro.aws.ec2;
+
+import gyro.core.Type;
+import software.amazon.awssdk.services.ec2.Ec2Client;
+import software.amazon.awssdk.services.ec2.model.PlacementGroup;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Query Placement Groups.
+ *
+ * Example
+ * -------
+ *
+ * .. code-block:: gyro
+ *
+ *    pg: $(external-query aws::placement-group {group-name: "abc", state: "available", strategy: "spread" })
+ */
+
+@Type("placement-group")
+public class PlacementGroupFinder extends Ec2TaggableAwsFinder<Ec2Client, PlacementGroup, PlacementGroupResource> {
+
+    private String groupName;
+    private String state;
+    private String strategy;
+
+    /**
+     * The name identifying the Placement Group.
```

Comment:
To be consistent with our existing docs. So that when an end user reads our documentation , it feels like one person wrote it.
```suggestion
     * The name of the Placement Group.
```